db/migrate/20210813215832_create_topics.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTopics < ActiveRecord::Migration[6.1]
+  def change
+    create_table :topics do |t|
+      t.string :label
```

Comment:
You can add the same validation here as in the model.

```suggestion
      t.string :label, null: false, unique: true
```